pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -730,72 +716,73 @@ func (r *RepoSyncReconciler) mapSecretToRepoSyncs(secret client.Object) []reconc
 
 func (r *RepoSyncReconciler) mapConfigMapToRepoSyncs(obj client.Object) []reconcile.Request {
 	objRef := client.ObjectKeyFromObject(obj)
-	if objRef.Namespace == v1.NSConfigManagementSystem {
-		// ignore ConfigMaps in the config-management-system namespace
+
+	// Use annotations/labels to map ConfigMap copies in config-management-system
+	if objRef.Namespace == configsync.ControllerNamespace {
```

Comment:
This will add labels and annotations to all ConfigMaps in the c-m-s namespace. What about ConfigMaps not for RepoSync Helm values?

Reply:
mapConfigMapToRepoSyncs doesn't add any labels. It just uses the labels to look up which RepoSync to trigger a Reconcile for. If the labels are not present, it doesn't trigger a Reconcile.

Reply:
Ideally, this will never happen, because the objects are immutable, but it's still possible for them to be deleted and/or re-created. If they're deleted and the last known object had our labels, then this will catch it and trigger Reconcile to re-create it.

Reply:
I see. It enqueued ConfigMaps with the SyncName and SyncNamespace labels and annotations.